lua/orgmode/objects/calendar.lua

```diff
@@ -46,13 +44,17 @@ function Calendar.new(data)
   this.on_day = data.on_day
   if data.date then
     this.date = data.date
-    this.month = this.date:set({ day = 1 })
   else
-    this.month = Date.today():start_of('month')
+    this.date = Date.today()
   end
   return this
 end
 
+---@return OrgDate
+function Calendar:month()
```

Comment:
I don't see this being used anywhere. Why we need it?

Reply:
I first use it to differentiate beween the date state and the information about the current month during the refactoring. But from the latter nothing is left anymore, so it got stale, but I didn't recognize it. Removed.